.frigg.yml

```diff
@@ -2,7 +2,8 @@ setup_tasks:
  - npm install
 
 tasks:
- - npm uninstall react-dom
+ - npm run lint
+ - npm test
+ - npm uninstall react-dom react-addons-test-utils
  - npm install react@0.13 && npm test
- - npm install react@latest react-dom@latest && npm test
  - npm run lint
```

Comment:
Good idea, it looks like `npm run lint` is there two times. Bad merge maybe? We can just keep it as the first task instead of last.